dissect/target/plugins/os/windows/sru.py

```diff
@@ -322,7 +340,7 @@
 }
 
 
-def transform_app_id(value):
+def transform_app_id(value: bytes | str | None) -> Union[bytes, str, None]:
```

Comment:
You're mixing the newer `|` syntax with the older `Union[]` syntax here. I think it's best to stick with `Union[]` for now. We might migrate entirely to the `|` syntax at some point.